content/version/2/1/code_of_conduct.gl.md

```diff
@@ -0,0 +1,133 @@
++++
+version = "2.1"
+aliases = ["/version/2/1"]
++++
+
+# Código de Conduta convido para contribuintes
```

Comment:
O de "convido" sóame forzadísimo.

Revisando outras traducións, penso que podemos poñer algo máis natural, como fan en portugués ("Pacto de código de conduta para colaboradores") ou mesmo máis simple, como fan no portugués de Brasil ("Código de conduta de colaboración")

Contribuínte existe en galego, mais penso que ten certa connotación fiscal, e existindo tamén como sinónimo contribuidor/a, penso que optaría pola segunda. Dito isto, ao ser unha palabra que marca o xénero, optar polo nome común "contribución" é máis inclusivo, pero novamente "contribución" neste contexto soa raro, mentres que "colaboración" soa mellor.

En resumidas contas, que eu titularía algo como "Código de Conduta para persoas colaboradoras" ou "Código de Conduta para colaboración" ou mesmo "Código de Conduta para participación".